### PR TITLE
Update: Modchat and Modjoin help commands

### PR DESCRIPTION
**Pull request by JetOU777**

### CODE REVIEW

**Comment by panpawn** [comment]
>The commit summary should not end in a period.

**Comment by Zarel** [comment]
I don't think we use the player symbol in help commands?

**Comment by JetOU777** [comment]
Zarel, it is used in publicroomhelp and hiddenroomhelp though

**Comment by Zarel** [comment]
Hm, the lists are getting unwieldy, maybe we should just show "and above"...

**Comment by panpawn** [comment]
"or higher" might sound better?

**Comment by Zarel** [comment]
`or above` sounds better to me, but I'm not strongly opinionated.

**Comment by AustinXII** [comment]
I think "or higher" is more commonly used but "or above" is also fine

**Comment by Zarel** [comment]
Sure, go with "or higher".

File: chat-plugins/roomsettings.js [hunk 1]
```diff
@@ -247,7 +247,7 @@ exports.commands = {
 			Rooms.global.writeChatRoomData();
 		}
 	},
-	modchathelp: ["/modchat [off/autoconfirmed/+/%/@/*/#/&/~] - Set the level of moderated chat. Requires: *, @ for off/autoconfirmed/+ options, # & ~ for all the options"],
+	modchathelp: ["/modchat [off/autoconfirmed/+/%/@/*/\u2606/#/&/~] - Set the level of moderated chat. Requires: *, @, \u2606 for off/autoconfirmed/+ options, # for off/autoconfirmed/+/%/@/*/\u2606/# options, & ~ for all the options"],
```
Comment (Zarel) [inline chat-plugins/roomsettings.js]: We definitely don't care about the fact that `#` can't set `modjoin &` or something. That's a tiny detail that doesn't belong in help text.

**Pull request closed without merging**